### ISSUE TITLE
bin/eslint-unused-modules does not check .ts files

### ISSUE THREAD

**Issue by muan** [github/eslint-plugin-github#89]
https://github.com/github/eslint-plugin-github/blob/master/bin/eslint-unused-modules.js

All the exports in our new TS files are currently unchecked. 😬

**Comment by koddsson** [comment]
I think this is because eslint defaults to files with a `.js` extension. Since our script takes in file globs we could just pass globs that include the `.ts` extension as a `bin/eslint-unused-modules src/**/*.ts`.

I looked into this a bit though and was hoping we could drop our custom solution for something built into TypeScript. I didn't find anything build into TypeScript but I did find [this gist](https://gist.github.com/dsherret/0bae87310ce24866ae22425af80a9864) that seems to yield correct results. I also timed the gist against our solution and the gist clocks in at around 13 seconds whereas our solution takes 50 seconds.
